setup.cfg

```diff
@@ -76,9 +76,8 @@ markers =
 
 [isort]
 multi_line_output = 3
```

Comment:
I think this is also for black compatibility but I4ve done that a long time ago

```suggestion
```

Reply:
I tried it. Result of `pre-commit run --all-files` are the same, so I removed it as well...